src/components/auth/Auth.js

```diff
@@ -46,96 +45,80 @@ class Auth extends React.Component<Props> {
   handleNavigatePrivacyPolicy = () => this.props.screenProps.push('PrivacyPolicy')
 
   render() {
-    const { styles, ...rest } = this.props
-    const {
-      acceptTermsLink,
-      acceptTermsText,
-      bottomContainer,
-      buttonLayout,
-      paragraph,
-      signInLink,
-      title,
-      topContainer,
-      wrapper,
-    } = styles
+    const { styles } = this.props
     return (
-      <View style={wrapper} {...rest}>
-        <View style={topContainer}>
-          <Title style={title}>Just a heads up!</Title>
-          <Description style={paragraph}>
-            {`All tokens in the Alpha are "test tokens".\nThey have NO real value.\nThey will be deleted at the end of the Alpha.`}
-          </Description>
-        </View>
-        <View style={bottomContainer}>
-          <Text style={acceptTermsText}>
-            {`By clicking the 'Create a wallet' button, you are accepting our `}
-            <LinkButton style={acceptTermsLink} onPress={this.handleNavigateTermsOfUse}>
+      <Wrapper backgroundColor="#fff" style={styles.mainWrapper}>
+        <Section justifyContent="space-between" style={styles.mainSection}>
+          <Wrapper style={styles.containerPadding}>
+            <Section.Row alignItems="center" justifyContent="center" style={styles.topRow}>
+              <Section.Text color="surface" fontFamily="slab" fontSize={22}>
+                {`Alpha tokens are \n for test use only!`}
+              </Section.Text>
+            </Section.Row>
+            <Section.Row alignItems="center" justifyContent="center" style={styles.bottomRow}>
+              <Section.Text color="surface" fontFamily="medium" fontSize={16}>
+                {`They have NO real value. \n And will be deleted at the end of the Alpha.`}
+              </Section.Text>
+            </Section.Row>
+          </Wrapper>
+        </Section>
+        <View style={styles.bottomContainer}>
+          <Text fontFamily="regular" fontSize={12} color="gray80Percent">
+            {`By clicking the 'Create a wallet' button,\n you are accepting our\n`}
+            <Section.Text style={styles.acceptTermsLink} onPress={this.handleNavigateTermsOfUse}>
```

Comment:
for consistency we should keep using `Text` in this context, as there's not a parent `Section` for these `Section.Text` elements.

Reply:
Done